tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -28,12 +28,19 @@
 !define WEBVIEW2INSTALLERPATH "{{webview2_installer_path}}"
 !define UNINSTKEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCTNAME}"
 !define MANUPRODUCTKEY "Software\${MANUFACTURER}\${PRODUCTNAME}"
+!define COMPRESSION "{{compression}}"
 
 Name "${PRODUCTNAME}"
 BrandingText "{{copyright}}"
 OutFile "${OUTFILE}"
 Unicode true
-SetCompressor /SOLID lzma
+
+; Set the compression algorithm. Default is LZMA.
+!if "${COMPRESSION}" == ""
+  SetCompressor /SOLID lzma
+!else
+  SetCompressor /SOLID "${COMPRESSION}"
+!endif
```

Comment:
I just found out that we still need this condition.

If the user doesn't provide `nsis` configuration in `tauri.config.json`, the bundler will not run the builder logic in `nsis.rs`, which means the `{{compression}}` won't be assigned a default value (`NsisCompression::Lzma`) hence `"${COMPRESSION}" == ""`

Reply:
Right! Let's leave it like that then